### PR TITLE
Es6 lib formatters

### PR DESCRIPTION
**Pull request by nicl**
Note, I want to remove the use of the default export but es lint is stopping me! (Says `prefer default import` but maybe I've misunderstood?)

## What does this change?

ES6 formatters lib module.

## What is the value of this and can you measure success?

ES6.

### CODE REVIEW

File: static/test/javascripts-legacy/spec/lib/formatters.spec.js [hunk 1]
```diff
@@ -1,25 +0,0 @@
...
-                [12345, '12,345'],
-                [123456, '123,456'],
-                [1234567, '1,234,567'],
-                [12345678, '12,345,678'],
-                [123456789, '123,456,789'],
-                [1234567890, '1,234,567,890']
-            ];
-            for (var i = 0; i < tests.length; i++) {
-                expect(formatters.integerCommas(tests[i][0])).toBe(tests[i][1]);
-                expect(formatters.integerCommas(tests[i][0]).toString()).toBe(tests[i][1]);
```
Comment (nicl) [inline static/test/javascripts-legacy/spec/lib/formatters.spec.js]: I couldn't see the value of this line, especially now that we think the return must be string or void, but maybe I missed something?

File: static/src/javascripts/lib/formatters.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        for (i = digits.length - 1; i >= 1; i -= 1) {
+            if ((len - i) % 3 === 0) {
+                digits.splice(i, 0, ',');
+            }
+        }
+        return digits.join('');
+    }
+};
+
+export default { integerCommas };
```
Comment (SiAdcock) [inline static/src/javascripts/lib/formatters.js]: I think we agreed to dispense with `default` exports

Comment (nicl) [inline static/src/javascripts/lib/formatters.js]: @SiAdcock yep! @sndrs has helped me overcome some lint woes here to make this happen :)

**Review by SiAdcock — APPROVED** [review]
Nice

**Pull request merged**